app/models/imports/podcast_timings_import.rb

```diff
@@ -47,6 +47,7 @@ def has_episode_with_guid?(guid)
   def validate_timings
     return errors.add(:timings, :blank) if timings.blank?
     return errors.add(:timings, :not_csv) if csv.blank? || csv[0].count < 2
+    return errors.add(:timings, :embedded_newlines) if csv.any? { |row| has_newlines?(row) }
```

Comment:
Was seeing some errors on imports containing unclosed quotes and/or newlines.  This just rejects any file that looks funny upfront, and prompts the user to remove columns like "title / description" if they have weird quoting.